docs/tutorials/kubernetes-cluster-in-gpc-with-gardener/_index.md

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
Type in the dir name - `kubernetes-cluster-in-gpc-with-gardener` should be `kubernetes-cluster-in-gcp-with-gardener/_index.md`.

```
mv ./docs/tutorials/kubernetes-cluster-in-gpc-with-gardener ./docs/tutorials/kubernetes-cluster-in-gcp-with-gardener/
```